pages/docs/getting-started.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: "Try Grouparoo Now!"
+date: "2021-02-02"
+section: "getting-started"
+pullQuote: "Get up and running with Grouparoo in just a few minutes to see what it's all about!"
+---
+
+Welcome! We're so glad you've chosen to spend some time with Grouparoo. See below for getting started with the basics.
+
+## Installation
+
+There are two ways to quickly install Grouparoo. For more options and for information on deploying to a production environment, see [the Installation & Deployment section](/docs/deployment).
+
+### Deploy to Heroku
+
+The simplest way to see Grouparoo in action is to deploy it to Heroku for free:
+
+[![Deploy to Heroku](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/grouparoo/app-example)
+
+### Run locally with Docker & Docker Compose
+
+If you have [Docker](https://www.docker.com/) installed, you can also run Grouparoo locally:
+
+    # in a new directory
+    curl -L https://www.grouparoo.com/docker-compose --output docker-compose.yml
+    docker-compose up
+
```

Comment:
Now that you don't need redis or postgres, do we want to keep the NPM way of trying out Grouparoo?

```
npx grouparoo init .
npm start
```